lib/src/node.dart

```diff
@@ -41,8 +41,11 @@ void _render(
     var indentWidth = indentWidthValue is int
         ? indentWidthValue
         : int.parse(indentWidthValue.toString());
+    var linefeed = parseLineFeed(options.linefeed);
```

Comment:
We should consistently capitalize `lineFeed`, at least in places where we it doesn't need to be identical to the JS API. I think capitalizing "feed" is more correct.